documentation/book/assembly-upgrade.adoc

```diff
@@ -3,20 +3,83 @@
 // master.adoc
 
 [id='assembly-upgrade-{context}']
-= {ProductName} and Kafka upgrades
+= Upgrade
 
-Each version of {ProductName} supports a range of versions of Apache Kafka.
-You can upgrade to a higher Kafka version as long as that version is supported by your version of {ProductName}.
+You can upgrade {ProductName} with no cluster downtime. Each version of {ProductName} supports a range of versions of Apache Kafka. You can upgrade to a higher Kafka version as long as that version is supported by your version of {ProductName}.
 In some cases, you can also downgrade to a lower supported Kafka version.
 
-When a newer version of {ProductName} is available, it may provide support for newer versions of Kafka. 
-Therefore, you will need to upgrade to the new version of {ProductName} before you can upgrade to a higher supported Kafka version.
-Upgrading the version of {ProductName} is done by upgrading the Cluster Operator deployment to the new version.
+Later versions of {ProductName} provide support for newer versions of Kafka, so you need to upgrade {ProductName} _before_ you can upgrade to a higher supported Kafka version.
```

Comment:
This sound strange. These are docs for version X.Y. Not for any older versions. So I do not think we should use things like _Later versions_.

Reply:
Changed to: '{ProductName} provides support for newer versions of Kafka, but you need to upgrade {ProductName} _before_ you can upgrade to a higher supported Kafka version.'